extension/httpfs/src/httpfs.cpp

```diff
@@ -113,8 +113,8 @@ void HTTPFileInfo::initialize() {
             // LCOV_EXCL_STOP
         }
     }
-    if (httpConfig.cacheFile) {
-        cachedFileInfo = hfs->getCachedFileManager().getCachedFileInfo(path);
+    if (httpConfig.cacheFile && length > CachedFileManager::MAX_SEGMENT_SIZE * 5) {
```

Comment:
Should we still do this optimization 😅 ? `length > CachedFileManager::MAX_SEGMENT_SIZE * 5`

Reply:
removed